.travis.yml

```diff
@@ -21,8 +21,10 @@ before_install:
 
 install:
 - export LD_LIBRARY_PATH=/usr/local/lib
-- export PATH=$PATH:prebuilt/:prebuilt/fval/
 - export LD_PRELOAD=prebuilt/custom_freetype/lib/libfreetype.so
```

Comment:
This other line can also be removed now.
Please delete the `prebuilt/custom_freetype/` directory as well.